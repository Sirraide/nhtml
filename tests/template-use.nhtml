template! template

div[foo=bar]#id.class