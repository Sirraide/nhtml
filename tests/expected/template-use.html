<div>
    <div></div>
</div>
<div class='class' id='id' foo='bar'></div>
<div></div>
<div></div>
<div></div>
